languages/javascript/exercises/concept/array-analysis/example.js

```diff
@@ -0,0 +1,23 @@
+export function getCardIndex(array, card) {
```

Comment:
Please update this example file according to the changes to the tests file and stub file.